tests/test_TFNetworkLayer.py

```diff
@@ -1018,6 +1038,11 @@ def test_MergeDimsLayer_basic():
     _check_MergeDimsLayer(session, {"shape": (4, None, 7), "time_dim_axis": 2}, (2, 4, 3, 7), {"axes": "static"}, (None, 4 * 7), (2, 3, 4 * 7))
     _check_MergeDimsLayer(session, {"shape": (1, None), "time_dim_axis": 2, "feature_dim_axis": 1}, (2, 1, 4), {"axes": "except_batch"}, (None,), (2, 4))
 
```

Comment:
Add two empty lines before a function.

Reply:
Ah sorry, the PyCharm I use was still set to default hints.